x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_color_property.tsx

```diff
@@ -277,11 +290,11 @@ export class DynamicColorProperty extends DynamicStyleProperty {
     }
   }
 
-  renderLegendDetailRow({ isPointsOnly, isLinesOnly, symbolId }) {
+  renderLegendDetailRow({ isPointsOnly, isLinesOnly, symbolId }: LegendProps) {
     const { stops, defaultColor } = this._getColorStops();
     const breaks = [];
-    stops.forEach(({ stop, color }) => {
-      if (stop) {
+    stops.forEach(({ stop, color }: { stop: string | number | null; color: string }) => {
+      if (stop !== null) {
```

Comment:
Not required for converting to typescript but fixes a bug where stop of `0` would not be rendered in legend.